pgtrigger/runtime.py

```diff
@@ -27,14 +36,17 @@ def _is_concurrent_statement(sql):
     return sql.startswith("create") and "concurrently" in sql
 
 
+DJANGO_4_2_PLUS = django.VERSION >= (4, 2)
```

Comment:
Made this a constant to make it a little faster, since caller `_can_inject_variable` is run on every query